bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/src/org/eclipse/equinox/internal/p2/ui/sdk/scheduler/AutomaticUpdateScheduler.java

```diff
@@ -56,7 +62,7 @@ public class AutomaticUpdateScheduler implements IStartup {
 	private IUpdateChecker checker;
 
 	@Override
-	public void earlyStartup() {
+	public void handleEvent(Event event) {
```

Comment:
Shouldn't we check if automated updates are disabled **before** starting the job?